README.rst

```diff
@@ -82,6 +82,14 @@ Usage
  # Output a parsable format (for syntax checking in editors like Vim, emacs...)
  yamllint -f parsable file.yaml
 
+.. code:: yaml
+
+  # Using it as a pre-commit hook (update the sha variable with the release version that you want)
+  - repo: https://github.com/adrienverge/yamllint.git
+    sha: v1.8.1
+    hooks:
+      - id: yamllint
+
```

Comment:
Please add this in docs rather than in README: https://yamllint.readthedocs.io/

For example you could create a new "Integration with other software" section.